### PR TITLE
Fixed issue #963

### PR DESCRIPTION
**Pull request by gokberksahin**
Removed skipping upon tapping the text. But only removed sentence from English and Turkish. Is there a way to remove all of them without deleting manually?

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -69,10 +69,7 @@
     <item quantity="other">%d uploads</item>
   </plurals>
   <string name="categories_not_found">No categories matching %1$s found</string>
-  <string name="categories_skip_explanation">Add categories to make your images more discoverable on Wikimedia Commons.
-
-Start typing to add categories.
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml]: I don't think this should have been removed?

**Comment by nicolas-raoul** [comment]
I believe that modifying in the English file will automatically prompt translators to modify the strings in other languages :-)

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (gokberksahin) [reply app/src/main/res/values/strings.xml]: You're right, i'm fixing it now :)

**Comment by neslihanturan** [comment]
After review is done, we can merge this, @gokberksahin . Thanks for your contribution:)

**Comment by gokberksahin** [comment]
Will this be merged?

**Comment by nicolas-raoul** [comment]
Thanks! :-)

**Pull request merged**